utensor_cgen/api/export.py

```diff
@@ -42,3 +49,77 @@ def tflm_keras_export(
   if output_tflite_fname:
     with open(output_tflite_fname, 'wb') as fid:
       fid.write(tflm_model_content)
+
+def pytorch_onnx_export(
+  model_or_path,
+  representive_dataset,
+  model_name=None,
+  optimizations=None,
+  config_file='utensor_cli.toml',
+  target='utensor',
+  output_onnx_fname=None
+) :
+  with tempfile.TemporaryDirectory(prefix='utensor_') as tmp_dir:
+    dir_path = Path(tmp_dir)
+    if isinstance(model_or_path, str):
+      model = torch.load(model_or_path)
+    elif isinstance(model_or_path, torch.nn.Module):
+      model = model_or_path
+    else:
+      raise RuntimeError(
+        "expecting a pytorch model or a path to torch model, get {}".format(
+          model_or_path
+        )
+      )
+
+    onnx_path = os.path.join(dir_path,f"{model_name}.onnx")
+    
+    torch.onnx.export(model, 
+      representive_dataset, 
+      onnx_path, 
+      verbose=True
+    )
+
+    # Print a human readable representation of the graph
+    onnx_model = onnx.load(onnx_path)
+    onnx.checker.check_model(onnx_model)
```

Comment:
Maybe it's also better if the export api only print human readable representation if the `verbose` argument is `True`.